js/webp-fix.js

```diff
@@ -0,0 +1,37 @@
+/*
+ * webp-fix.js
+ *
+ *	Detects lack of webp support, loads full images instead. 
```

Comment:
Specifying that this is for WebP _thumbnails_ would make it clearer. It's possible to allow .webp as an upload format instead.

Reply:
If possible, maybe changing the filename to make it clear it's for webp-thumbnail workaround.